assignments/week-1/trees.k

```diff
@@ -0,0 +1,62 @@
+function turnRight() {
+	repeat(3) {
+		turnLeft();
+	}
+}
+
+
+function moveThree() {
+	repeat(3) {
+		move();	
+	}
+}
+
+function moveTwo() {
+	repeat(2) {
+		move();
+	}
+}
+
+function treeTops() {
+	putBeeper();
+	move();
+	putBeeper();
+	turnRight();
+	move();
+	putBeeper();
+	turnRight();
+	move();
+	putBeeper();
+	
+}
+
+function firstlastTree() {
+	turnLeft();
+	moveThree();
+	treeTops();
+	moveThree();
+	turnLeft();
+	moveTwo();
+}
+
+
+
+function main() {
+	move();
```

Comment:
I think you can use `while` loops together with the different `blocked?` checks to avoid hard-coding at each point how many steps you have to move in whatever direction.